src/NzbDrone.Common/Disk/OsPath.cs

```diff
@@ -71,6 +71,11 @@ private static string FixSlashes(string path, OsPathKind kind)
                 case OsPathKind.Windows:
                     return path.Replace('/', '\\');
                 case OsPathKind.Unix:
+                    path = path.Replace('\\', '/');
+                    while (path.Contains("//"))
```

Comment:
The while is unnecessary.

The return line also doesn’t make sense given the previous line.

Reply:
I added the requested test and fixed the return vallue.
I use the while loop for the case /a///path//////somewhere. In the case such a path is provided.